src/Factories/FunFunFactory.php

```diff
@@ -598,7 +600,24 @@ public function newAddSubscriptionJsonPresenter(): AddSubscriptionJsonPresenter
 	}
 
 	public function newGetInTouchHtmlPresenter(): GetInTouchHtmlPresenter {
-		return new GetInTouchHtmlPresenter( $this->getLayoutTemplate( 'contact_form.html.twig' ), $this->getTranslator() );
+		return new GetInTouchHtmlPresenter(
+			$this->getLayoutTemplate( 'contact_form.html.twig' ),
+			$this->getTranslator(),
+			$this->getGetInTouchCategories()
+		);
+	}
+
+	public function getGetInTouchCategories(): array {
+		try {
+			$json = ( new SimpleFileFetcher() )->fetchFile( $this->getI18nDirectory() . '/data/contact_categories.json' );
+			if ( $json === "" ) {
+				throw new RuntimeException();
```

Comment:
Please add an exception description like "Could not load categories for contact form.", it will be easier to debug when looking at a log file.